explorer/explorerroutes.go

```diff
@@ -1615,8 +1640,18 @@ func (exp *explorerUI) Search(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Split searchStr to the first part corresponding to transaction hash and
+	// to the second part corresponding to transaction output.
+	searchStrSplit := strings.Split(searchStr, ":")
+	var searchStrRewritten string
+	if len(searchStrSplit) >= 2 {
```

Comment:
A pattern that I tend to prefer is to eliminate `else` branches if the variable initialization can handle the case.  That is,

```go
searchStrRewritten := searchStrSplit[0]
if len(searchStrSplit) > 1 {
	// Validate searchStrSplit[1]
	...
	searchStrRewritten = searchStrSplit[0] + "/out/" + searchStrSplit[1]
}
```

Although I would also want to throw an error if `len(searchStrSplit) > 2`, which would indicate an invalid path, so you might consider a switch on `len(searchStrSplit)` instead.